samples/cpp/hello_nv12_input_classification/README.md

```diff
@@ -10,65 +10,68 @@ The following C++ API is used in the application:
 | Infer Request Operations | `ov::InferRequest::set_tensor`, `ov::InferRequest::get_tensor` | Operate with tensors |
 | Preprocessing | `ov::preprocess::InputTensorInfo::set_color_format`, `ov::preprocess::PreProcessSteps::convert_element_type`, `ov::preprocess::PreProcessSteps::convert_color` | Change the color format of the input data |
 
-Basic OpenVINO™ Runtime API is covered by [Hello Classification C++ sample](../hello_classification/README.md).
+Basic OpenVINO™ Runtime API is described in [Hello Classification C++ sample](../hello_classification/README.md).
 
 | Options | Values |
 | :--- |:--- |
 | Validated Models | [alexnet](@ref omz_models_model_alexnet) |
-| Model Format | OpenVINO™ toolkit Intermediate Representation (\*.xml + \*.bin), ONNX (\*.onnx) |
-| Validated images | An uncompressed image in the NV12 color format - \*.yuv |
+| Model Format | OpenVINO Intermediate Representation (\*.xml + \*.bin), ONNX (\*.onnx) |
+| Validated images | An uncompressed image in the `NV12` color format - \*.yuv |
 | Supported devices | [All](../../../docs/OV_Runtime_UG/supported_plugins/Supported_Devices.md) |
 | Other language realization | [C](../../../samples/c/hello_nv12_input_classification/README.md) |
 
 ## How It Works
 
-At startup, the sample application reads command line parameters, loads the specified model and an
-image in the NV12 color format to an OpenVINO™ Runtime plugin. Then, the sample creates an synchronous inference request object. When inference is done, the application outputs data to the standard output stream. You can place labels in .labels file near the model to get pretty output.
+At startup, the sample application reads command-line parameters, loads the specified model and an
+image in the `NV12` color format to an OpenVINO Runtime plugin. Then, the sample creates an synchronous inference request object. When inference is done, the application outputs data to the standard output stream. You can place labels in `.labels` file near the model to get pretty output.
```

Comment:
Then, the sample creates a synchronous inference request object.

Reply:
Done in https://github.com/openvinotoolkit/openvino/pull/14197/commits/8397872e5f1da428cc20134d62925dfc851e40c0